src/popup/views/Promotions.svelte

```diff
@@ -64,8 +64,8 @@
 </script>
 
 <hr class="mt-6" />
-<footer class="mt-5">
-  <div>{i18n.labelSubheader}</div>
+<details class="mt-5">
+  <summary class="mb-1 ml-1 text-secondary cursor-pointer">&nbsp; {i18n.labelSubheader}</summary>
```

Comment:
Though I'm not sure what's with the `&nbsp;`

Reply:
I prefer to have a margin instead

Reply:
replaced with margin

Reply:
replaced with margin

Reply:
GitLens decided to use my old work account today 💀